packages/loot-core/src/platform/server/sqlite/index.electron.ts

```diff
@@ -84,7 +84,12 @@ export async function asyncTransaction(db, fn) {
 }
 
 export function openDatabase(pathOrBuffer) {
-  return new Database(pathOrBuffer);
+  let db = new Database(pathOrBuffer);
+  db.function('regexp', { deterministic: true }, (regex, text) => {
+    return new RegExp(regex, 'ui').test(text) ? 1 : 0;
```

Comment:
I'm not 100% certain on what these database functions are used for but they probably similarly need to have `text === null` checks.